tensorflow/contrib/cmake/external/protobuf.cmake

```diff
@@ -14,38 +14,46 @@
 # ==============================================================================
 include (ExternalProject)
 
-set(PROTOBUF_INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/protobuf/src/protobuf/src)
-set(PROTOBUF_URL https://github.com/google/protobuf.git)
-set(PROTOBUF_TAG b04e5cba356212e4e8c66c61bbe0c3a20537c5b9)
+if (tensorflow_PROTOBUF_PROVIDER STREQUAL module AND tensorflow_GRPC_PROVIDER STREQUAL module)
+  set(PROTOBUF_INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/protobuf/src/protobuf/src)
+  set(PROTOBUF_URL https://github.com/mrry/protobuf.git)  # Includes MSVC fix.
```

Comment:
Minor tweak here: replace `mrry` with `google` to fix the build error.

Reply:
Updated, hope it helps.